src/ocean_component.cpp

```diff
@@ -235,6 +218,46 @@ void OceanComponent::prepareToRun() {
     const double I_volume = ocean_area * thick_inter;
     const double D_volume = ocean_area * thick_deep;
 
+    // Calculate the fraction of volume of the two surface boxes, the deep, and
+    // intermediate ocean boxes. The fraction of the volumes will be used to determine the
+    // inital size of the preindustrial carbon pools for Hector's ocean boxes.
+    const double LL_vol_frac = LL_volume / (LL_volume + HL_volume);
+    const double HL_vol_frac = 1 - LL_vol_frac;
+    const double I_vol_frac = I_volume / (I_volume + D_volume);
+    const double D_vol_frac = 1 - I_vol_frac;
+    
+    // Define the size of the preindustrial ocean surface and intermediate-deep ocean carbon pools
+    // from IPCC AR6 Figure 5.12.
+    const double preind_C_surface = 900;   // (Pg C) IPCC AR6 Figure 5.12
+    const double preind_C_ID = 37100;      // (Pg C) IPCC AR6 Figure 5.12
+    
+    // Partition the preindustrial ocean carbon pools by volume.
+    const double LL_preind_C = LL_vol_frac * preind_C_surface;
+    const double HL_preind_C = HL_vol_frac * preind_C_surface;
+    const double I_preind_C = I_vol_frac * preind_C_ID;
+    const double D_preind_C = D_vol_frac * preind_C_ID;
+
+
+    // Set up our ocean box model.
+    H_LOG( logger, Logger::DEBUG ) << "Setting up ocean box model" << std::endl;
+   // surfaceHL.initbox( HL_preind_C, "HL" );
+       surfaceHL.initbox( unitval( 140, U_PGC ), "HL" );
```

Comment:
😕 Why are we computing the preindustrial values above and then plugging in 140, etc. here?

Reply:
gah! good catch those are the old values being where as `// surfaceHL.initbox( HL_preind_C, "HL" );` is the code we want to commit but mistakenly pushed a version I was using to debug.